paramiko/hostkeys.py

```diff
@@ -151,31 +175,31 @@ def __len__(self):
                 return len(self.keys())
 
             def __delitem__(self, key):
-                for e in list(self._entries):
-                    if e.key.get_name() == key:
-                        self._entries.remove(e)
+                for entry in list(self._entries):
+                    if entry.key.get_name() == key:
+                        self._entries.remove(entry)
                 else:
                     raise KeyError(key)
 
             def __getitem__(self, key):
-                for e in self._entries:
-                    if e.key.get_name() == key:
-                        return e.key
+                for entry in self._entries:
+                    if entry.key.get_name() == key:
+                        return entry.key
```

Comment:
These diff sections just renaming `e` to `entry` is not really needed in this PR IMHO.